ci/setup_env.sh

```diff
@@ -64,15 +64,9 @@ log_info "Release branch(es) is(are): ${release_branch}/*"
 log_info "Github tag is: ${github_tag}"
 
 # Checking if it is a release build
-if git branch -r --contains "${github_tag}" | grep -xqE ". origin\/${release_branch}/[^/]+$"; then
+if git branch -r --contains "${github_tag}" | grep -xqE ". origin\/${release_branch}/${github_tag}"; then
```

Comment:
The new desired approach here github tag deriving from `release/github_tag` branch? Therefore we consider release ONLY if tag = 3.3.3 for example and it MUST derive from `release/3.3.3` branch?

If this is the desired approach then everything looks good

Reply:
Yes, just to make a couple examples more:

- tag `3.3.3-3.3.3`, branch `release/3.3.3-3.3.3`,
- tag `3.3.3-3.3.3-rc1`, branch `release/3.3.3-3.3.3-rc1`,
- tag `3.3.3-3.3.3-temp`, branch `release/3.3.3-3.3.3-temp`.

This modification comes mainly from the following non-working scenario we had before:

- tag `3.3.3-rc1` (created at commit sha `abcd`), branch `release/3.3.3-rc1`,
- tag `3.3.3` (created at commit sha `abcd`), branch `release/3.3.3`,
- falling into error case `WARNING: More than 1 GitHub '${release_branch}/*' branch contains current GitHub tag: ${github_tag}. The build is not going to be released ...`.